### DISCUSSION TITLE
Chipsee AIO with crankshaft

### DISCUSSION THREAD

**Discussion by AlexanderYW** [opencardev/crankshaft#520] [Q&A]
Hi fellow Developers,

Anyone who can guide me how to run crankshaft on a [chipsee AIO with CM4](https://chipsee.com/product/aio-cm4-101/) or can tell me how to?, all i get is a black screen when it boot, i tried the to edit the config.txt and cmdline.txt files and make them match whats provided by Chipsee's with [image](https://github.com/Chipsee/industrial-pi) (source can be viewed in different branch) which is a modified Raspbian image 

I'm doing a DIY project where I add the Chipsee AIO as a headunit in my car, it's an old Mitsubishi Lancer from 2007 :D

**Comment by matt2005** [comment]
Does Debian buster work on it?
